.typos.toml

```diff
@@ -0,0 +1,13 @@
+# Configuration for the typos spell checker utility (<https://github.com/crate-ci/typos>).
+
+[files]
+extend-exclude = [
+    # "uefi/src/table/boot.rs"
+]
+
+[default.extend-words]
+# FOOBAR = "FOOBAR"
+
+[default.extend-identifiers]
+# FOOBAR = "FOOBAR"
+
```

Comment:
Are these default values? If so, can we drop the config file entirely?

Reply:
It might happen that we have to add something there eventually. This is also necessary in a few other projects of mine. I thought, it might be easier for us if we already have the file as template in the repo. What do you think?